### PR TITLE
add hivemind cad + port to gallery

### PR DESCRIPTION
**Pull request by polargh**

### CODE REVIEW

File: source/docs/appendix/gallery.rst [hunk 1]
```diff
@@ -8,6 +8,12 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
 2023-2024 Center Stage
 ------------------------
 
+23396 hivemind
+^^^^^^^^^^^^^^^^^^^^^
+
+- `CAD <https://go.hivemindrobotics.net/cad>`__
+- `Portfolio <https://go.hivemindrobotics.net/csportfolio>`__
```
Comment (abidingabi) [inline source/docs/appendix/gallery.rst:15]: Please do not use link shorteners, or your own hosting for your portfolio—we would rather link to somewhere that we know will last.

File: source/docs/appendix/gallery.rst [hunk 2]
```diff
@@ -8,6 +8,12 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
 2023-2024 Center Stage
 ------------------------
 
+23396 hivemind
```
Comment (abidingabi) [inline source/docs/appendix/gallery.rst:11]: The teams within a given season are sorted in numerical order, and so hivemind should be last out of the teams present for Center Stage.

**Review by abidingabi — CHANGES_REQUESTED** [review]

File: source/docs/appendix/gallery.rst [hunk 3]
```diff
@@ -26,6 +20,12 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
 - `Portfolio (English) <https://drive.google.com/file/d/1wx9naEp4gLObyNXjph1JLryPXUEph6ik/view>`__
 - `Portfolio (Romanian) <https://drive.google.com/file/d/1lNLNuKn5PMewyk5oaLIF3951BtJYJSvq/view>`__
 
+23396 hivemind
+^^^^^^^^^^^^^^^^^^^^^
+
+- `CAD <https://cad.onshape.com/documents/fba422c04d9f259e36433b8c/w/661ed7aa8a40b6c0f5d26203/e/6205d0d7691937476f4d9a4c?renderMode=0&uiState=662b07d69f2a5f750315222b>`__
+- `Portfolio <https://www.dropbox.com/scl/fi/djsasr2ofjwxe2u6x30q4/23396-CS.pdf?rlkey=i6i2m0biob18iiegup48wnkus&st=kntv960r&dl=0>`__
```
Comment (abidingabi) [inline source/docs/appendix/gallery.rst:27]:
```suggestion
- `CAD <https://cad.onshape.com/documents/fba422c04d9f259e36433b8c/w/661ed7aa8a40b6c0f5d26203/e/6205d0d7691937476f4d9a4c>`__
- `Portfolio <https://www.dropbox.com/scl/fi/djsasr2ofjwxe2u6x30q4/23396-CS.pdf>`__
```

File: source/docs/appendix/gallery.rst [hunk 4]
```diff
@@ -20,6 +20,12 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
...
```
Comment (abidingabi) [inline source/docs/appendix/gallery.rst:27]:
```suggestion
- `CAD <https://cad.onshape.com/documents/fba422c04d9f259e36433b8c/w/661ed7aa8a40b6c0f5d26203/e/6205d0d7691937476f4d9a4c>`__
- `Portfolio <https://www.dropbox.com/scl/fi/djsasr2ofjwxe2u6x30q4/23396-CS.pdf>`__
```

**Review by abidingabi — APPROVED** [review]

**Comment by ArushYadlapati** [comment]
wow its hivemmind

**Pull request merged**